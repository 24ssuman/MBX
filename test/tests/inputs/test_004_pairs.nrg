SYSTEM H2O
MOLECULE
MONOMER h2o
 O                 -5.00580722   -4.19860621    0.00000000
 H                 -4.04580722   -4.19860621    0.00000000
 H                 -5.32626181   -3.29367038    0.00000000
ENDMON h2o
ENDMOL
MOLECULE
MONOMER h2o
 O                 -1.35736739   -1.13113945    0.00000000
 H                 -0.39736739   -1.13113945    0.00000000
 H                 -1.67782198   -0.22620362    0.00000000
ENDMON h2o
ENDMOL
MOLECULE
MONOMER h2o
 O                 -7.84256716    0.82950227    0.00000000
 H                 -6.88256716    0.82950227    0.00000000
 H                 -8.16302175    1.73443810    0.00000000
ENDMON h2o
ENDMOL
MOLECULE
MONOMER h2o
 O                -57.46188630  -52.56027996    0.00000000
 H                -56.50188630  -52.56027996    0.00000000
 H                -57.78234089  -51.65534413    0.00000000
ENDMON h2o
ENDMOL
MOLECULE
MONOMER h2o
 O                -50.07177494  -52.56027996    0.00000000
 H                -49.11177494  -52.56027996    0.00000000
 H                -50.39222953  -51.65534413    0.00000000
ENDMON h2o
ENDMOL
MOLECULE
MONOMER h2o
 O                 67.26509525   56.17992621    0.00000000
 H                 68.22509525   56.17992621    0.00000000
 H                 66.94464067   57.08486204    0.00000000
ENDMON h2o
ENDMOL
ENDSYS
