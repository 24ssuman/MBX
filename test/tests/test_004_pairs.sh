#!/bin/bash

filename=$(basename "$0")
filename="${filename%.*}"

$CU_HOME/test/bin-test/dimers-test inputs/${filename}.nrg > output

if diff expected/${filename}.out output &> /dev/null ; then
  echo "TEST $filename PASSED" 
else
  echo "TEST $filename FAILED"
fi

rm -f CU.log output 
