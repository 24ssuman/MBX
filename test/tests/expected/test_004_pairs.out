Dimer[         0] ->        0       1
Dimer[         1] ->        0       2
Dimer[         2] ->        1       2
Dimer[         3] ->        3       4
